src/main/java/pl/ismop/web/client/dap/DapController.java

```diff
@@ -324,10 +322,7 @@ public void onSuccess(Method method, MeasurementsResponse measurementsResponse)
 	}
 
 	public void getLastMeasurements(List<String> timelineIds, Date date, final MeasurementsCallback callback) {
-		String until = converter.format(date);
-		String from = converter.format(new Date(date.getTime() - 86_400_000L));
-		measurementService.getLastMeasurements(merge(timelineIds, ","), from, until,
-				new MeasurementsRestCallback(callback));
+		measurementService.getLastMeasurements(merge(timelineIds, ","), new MeasurementsRestCallback(callback));
```

Comment:
Date is needed here. We don't want last, last measurement but last measurement before specific date.